src/main/java/Duke.java

```diff
@@ -1,10 +1,95 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    static int idx = 0;
+    static String[] list = new String[100];
+    static boolean[] marked = new boolean[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Arrays.fill(marked, Boolean.FALSE);
+        // String logo = " ____ _ \n"
+        // + "| _ \\ _ _| | _____ \n"
+        // + "| | | | | | | |/ / _ \\\n"
+        // + "| |_| | |_| | < __/\n"
+        // + "|____/ \\__,_|_|\\_\\___|\n";
+
+        Scanner input = new Scanner(System.in);
+        System.out.println("    _________________________________________");
+        System.out.println("    Hello I'm Duke\n    What can I help you with?");
+        System.out.println("    _________________________________________");
+        String ans = "";
+
+        String command = "";
+        int separator;
+        int index;
+        while (true) {
+            ans = input.nextLine();
+            separator = ans.indexOf(' ');
+            if (separator == -1) {
+                command = ans;
```

Comment:
I like how the single statement conditional is still wrapped by curly brackets. Good adherence to coding standards.